src/modules/auth/auth.service.ts

```diff
@@ -31,21 +32,14 @@ export class AuthService {
       const isMatch = await bcrypt.compare(password, user.password);
 
       if (!isMatch) {
-        this.logger.warn(`Password mismatch for user: ${username}`);
-        throw new HttpException(
-          'Invalid credentials.',
-          HttpStatus.UNAUTHORIZED,
-        );
+        throw new HttpException('Incorrect Password.', HttpStatus.UNAUTHORIZED);
       }
```

Comment:
During a password and email mismatch we througha mixed error, we never specify whether email was wrong or password.

This was better `Invalid credentials`